manifests/r/Roblox/RobloxStudio/1.0/Roblox.RobloxStudio.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.0

```

Comment:
```suggestion
PackageIdentifier: Roblox.RobloxStudio.Beta
PackageVersion: 1.6.0.29627
```

Reply:
in PackageIdentifier, I know you added .Beta because of the installer's name. Well, Roblox Studio is not in beta. So I will not add it.

Reply:
But I will fix PackageVersion cause i added 1.0 bcs it made spaces intead of dots so i thought it got it wrong.

Reply:
instead*

Reply:
> Well, Roblox Studio is not in beta.

I double-checked on their official forums, the newer version was made available to everyone, but they forgot to update the installer name.